test/cli/cli.test.js

```diff
@@ -9,8 +9,8 @@ describe("cli", () => {
         let rootDir = path.join(__dirname, "resources");
 
         return execFile(
```

Comment:
We might want to switch these to a plain `exec()` if we're calling `node` as the command - looks like `execFile` expects the first argument to be the file to execute!